modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/rest-openapi.yaml

```diff
@@ -12832,6 +12832,86 @@ paths:
                       - HeadlessCommerceAdminCatalog.write
             summary: Updates a sku by external reference code.
             tags: ["Sku"]
+    /skus/by-externalReferenceCode/{externalReferenceCode}/change-external-reference-code/{newExternalReferenceCode}:
+        put:
+            operationId: updateSkuExternalReferenceCodeToNewExternalReferenceCode
```

Comment:
Is this auto-generated?

Reply:
@alessiorendina no it is not, do I need to change something here?